atomic_reactor/util.py

```diff
@@ -1389,6 +1389,9 @@ def get_log_files(self, osbs: OSBS, pipeline_run_name) -> List[Output]:
             if task_platform in logfiles:
                 log_file = logfiles[task_platform]
             else:
+                logfiles['noarch'].write(bytearray(f'{task_platform} build has started. '
+                                                   + 'Check platform specific logs\n', 'utf-8'))
```

Comment:
Shouldn't we write this only if `task_platform != 'noarch'`?

Reply:
mmmm right it should

Reply:
yes, this will only occur for `!= 'noarch'` because `noarch` will already be in `logfiles`